BRANCHING.md

```diff
@@ -0,0 +1,71 @@
+# Mina Branching Policy
+
+- `master`: current stable release.
+  - It is frozen from the crypto side, does not depend on `proof-systems`.
+  - Never commit to it directly, except to introduce a hotfix.
+- `compatible`: scheduled to be released.
+  - It contains all the changes which are literally backwards compatible with what people currently run on the mainnet. Any nodes running a version of mina based off of compatible should connect to the current mainnet.
+- `rampup`: what is deployed on the testnet
+  - Compatible with `compatible`.
```

Comment:
This isn't true, `rampup` and `compatible` are mutually incompatible.

Reply:
Fixed.